### DISCUSSION TITLE
Woche 7 - Frontend, Frontend und noch mehr Frontend

### DISCUSSION THREAD

**Discussion by lukashert** [Chrissi-Ruege/HealthKeeper#17] [General]
Hallo zusammen!

In dieser Woche haben wir vor allem im Frontend einige Fortschritte zu zeigen. Neben dem Food Journal haben wir uns um unsere Login Seite gekümmert. 

## Food Journal 
In dieser Woche haben wir uns mit dem Frontend des Food Journal beschäftigt. Dieses wurde von Lukas Melcher implementiert.  Mithilfe der "Vorheriger Tag" und "Nächster Tag" Buttons kann zwischen den letzten Tagen hin und her gewechselt werden. Darunter kann man das Frühstück, MIttagessen und Abendessen einsehen und Speisen hinzufügen. Später sollen hier noch die jeweiligen Kalorien und Macronährstoffe angezeigt werden. Außerdem müssen die Daten noch im Backend gespeichert werden.

![food1](https://github.com/Chrissi-Ruege/HealthKeeper/assets/20227840/9286c9ef-0619-4cf3-b684-7ce9a51c4421)
![food2](https://github.com/Chrissi-Ruege/HealthKeeper/assets/20227840/cce42467-d9e3-4f88-91aa-d192999143da)

## Login Seite
Katja hat sich in dieser Woche mit dem Design der Login Seite beschäftigt. Hier muss später noch das Backend angebunden werden. 

<img width="914" alt="login" src="https://github.com/Chrissi-Ruege/HealthKeeper/assets/20227840/742a6b28-6ea5-4571-87c9-334b3dec34f0">

Viele Grüße,
euer HealthKeeper Team

**Comment by niklaskrauth** [comment]
Das FrontEnd sieht gut aus. Vielleicht kann man noch ein paar Bilder hinzufügen da der Hintergrund noch etwas leer aussieht.
